components/Icons.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Icon from "react-native-vector-icons/MaterialIcons";
```

Comment:
you can "MaterialIcons" instead of "Icon". in case we will want to use other icons in future